tests/units/classes/test.php

```diff
@@ -466,6 +479,22 @@ public function testGetTestedClassPath()
 			;
 		}
 
+		public function testCallStaticOnTestedClass()
+		{
+			$this
+				->if($this->setTestedClassName('\mageekguy\atoum\withStatic'))
+				->then
+					->string($this->callStaticOnTestedClass('staticMethod', $return = uniqid()))->isEqualTo($return)
```

Comment:
This test failed cause the `setTestedClassName` raise an (normal) exception.

Any clue to how i can test my new asserter ? ping @jubianchi @mageekguy